### ISSUE TITLE
Konva warning: methods are deprecated.

### ISSUE THREAD

**Issue by sid3r** [konvajs/vue-konva#115]
Hello, I've just implemented the transformer and it's giving me this warning:

`Util.js?9cb8:673 Konva warning: tr.setNode(shape), tr.node(shape) and tr.attachTo(shape) methods are deprecated. Please use tr.nodes(nodesArray) instead.`

the code works fine but it would be fix it, I'm using the code provided as example on website. the warning is emitted on these lines: 

`if (selectedNode) {
      // attach to another node
        transformerNode.attachTo(selectedNode);
      } else {
        // remove transformer
        transformerNode.detach();
      }`

**Comment by lavrton** [comment]
I updated docs.
